src/client/eppo-client.ts

```diff
@@ -501,12 +501,31 @@ export default class EppoClient {
     return { variation, action };
   }
 
+  getBestBanditAction(
+    flagKey: string,
+    subjectKey: string,
+    subjectAttributes: BanditSubjectAttributes,
+    actions: BanditActions,
+    defaultValue: string,
+  ): Omit<IAssignmentDetails<string>, 'evaluationDetails'> {
+    const { variation, action } = this.getBanditActionDetails(
```

Comment:
I wonder if we even need to do this. I don't think we want a variation assignment either (and the logging that comes with that). I think we just use our bandit references and pull the first (for now, there should only be one) bandit associated with the flag, forcing the use of the bandit. Although then we'd need to fold the attribute and model retrieval stuff into their own methods so it's not duplicated. I think its worth it though to ensure this call returns a bandit action vs. another variation (e.g., control)

Reply:
aaaand it's gone